Fomantic-UI install fails - tasks.js expects error.filename

Duplicate of this issue, carried over from Semantic-UI:
https://github.com/Semantic-Org/Semantic-UI/issues/2971

I'm getting this error in my GitLab-CI during installation of Fomantic-UI, due to which my build tasks fail:
```
TypeError: Cannot read property 'match' of undefined
     at DestroyableTransform.errorHandler (/builds/cataxcab/the-wall/node_modules/fomantic-ui/tasks/config/tasks.js:108:29)
     at DestroyableTransform.emit (events.js:203:15)
     at DestroyableTransform.EventEmitter.emit (domain.js:448:20)
   ...
   ...
   npm ERR! code ELIFECYCLE
   npm ERR! errno 1
   npm ERR! fomantic-ui@2.8.4 install: `gulp install`
   npm ERR! Exit status 1
   npm ERR! 
   npm ERR! Failed at the fomantic-ui@2.8.4 install script.
``` 
[Complete error link](https://gitlab.com/cataxcab/the-wall/-/jobs/551032356)

The corresponding line in tasks.js is:
```
if(error.filename.match(/theme.less/)) {
```
However, here, `filename` can be (and is) undefined.
The case where the property `filename` does not exist must be handled.

## Steps to reproduce
1. Set up a .gitlab-ci.yml task [like this](https://gitlab.com/cataxcab/the-wall/-/blob/5131b65919a33ca4bd20fee0e2e54fdb749392d0/.gitlab-ci.yml)
2. Run the Job

## Expected result
The job should succeed, Fomantic-UI should not face any errors

## Links
GitLab repo where the error is reproduced: https://gitlab.com/cataxcab/the-wall
[Corresponding issue in Semantic-UI](https://github.com/Semantic-Org/Semantic-UI/issues/2971)
[Complete error](https://gitlab.com/cataxcab/the-wall/-/jobs/551032356)
[My .gitlab-ci.yml file](https://gitlab.com/cataxcab/the-wall/-/blob/5131b65919a33ca4bd20fee0e2e54fdb749392d0/.gitlab-ci.yml)
[Corresponding question on SO](https://stackoverflow.com/questions/61780956/how-to-install-and-build-fomantic-ui-in-gitlab-ci)


## Version
**fomantic-ui** : 2.8.4
**npm** : 6.14.5
**electron** : 7.1.11

--- a/tasks/config/tasks.js
+++ b/tasks/config/tasks.js
@@ -105,7 +105,7 @@ module.exports = {
             theme,
             element
           ;
-          if(error.filename.match(/theme.less/)) {
+          if(error && error.filename && error.filename.match(/theme.less/)) {
             if (error.line == 9) {
               element = regExp.variable.exec(error.message)[1];
               if (element) {

